archive/restore.go

```diff
@@ -78,7 +78,7 @@ func importBlocks(chain blockchainInterface, blockStream *blockStream, progressi
 			return err
 		}
 
-		if err := chain.WriteBlock(nextBlock); err != nil {
+		if err := chain.WriteBlock(nextBlock, "restore"); err != nil {
```

Comment:
how about defining constant values for reason?

Reply:
Defined:

[e089b5c](https://github.com/0xPolygon/polygon-edge/pull/650/commits/e089b5c50d08a56ba2e091e317fc8e4ed964bbfe)